lib/tasks/sync_pull_requests_reputation.rake

```diff
@@ -1,5 +1,6 @@
 desc 'Sync pull requests reputation'
 task sync_pull_requests_reputation: :environment do
   Github::PullRequest::SyncRepos.()
+  Github::OrganizationMember::SyncMembers.()
   User::ReputationToken::AwardForPullRequests.()
 end
```

Comment:
I think this rake task should change to be `FetchAndSyncAllPullRequestsReputationJob.perform_later`.

Reply:
Logic being:
1. DRY
2. We can't create reputation on bastion anyway.

Reply:
Actually, if `User::ReputationToken::AwardForPullRequests` now creates jobs to create (which I think it does) then we could change it to `perform_now` instead, so we can run the bits we **can** run on bastion there.

Reply:
`User::ReputationToken::AwardForPullRequests` does indeed create jobs:

```
pull_requests.find_each do |pr|
  AwardReputationForPullRequestJob.perform_later(pr.data)
  ..
```